index.bs

```diff
@@ -493,15 +493,21 @@ This Section defines a number of new set of <a>Constrainable Properties</a> for
   <dd>This reflects the supported range of <a>contrast</a>. Values are numeric.  Increasing values indicate increasing contrast.</dd>
 
   <dt><dfn dict-member for="MediaTrackCapabilities"><code>pan</code></dfn></dt>
-  <dd>This reflects the <a>pan</a> value range supported by the UA and by the track.
+  <dd>
+    This reflects the <a>pan</a> value range supported by the UA and by the track.
 
-  If the track has been created without <a>requesting permission to use</a> (as defined in [[!permissions]]) a PermissionDescriptor with its name member set to {{PermissionName/camera}} and its {{CameraDevicePermissionDescriptor/panTiltZoom}} member set to true or if that permission request is denied, the track does not support <a>pan</a>.
-  In that case the UA MUST NOT expose the <a>pan</a> value range.
+    If the track has been created without <a>requesting permission to use</a> (as defined in [[!permissions]]) a PermissionDescriptor with its name member set to {{PermissionName/camera}} and its {{CameraDevicePermissionDescriptor/panTiltZoom}} member set to true or if that permission request is denied, the track does not support <a>pan</a>.
+    In that case the UA MUST NOT expose the <a>pan</a> value range but MAY provide an empty {{MediaSettingsRange}} dictionary to indicate that the underlying video source supports <a>pan</a>.
 
-  <div class="note">
-    It does not matter whether the algorithm to <a>request permission to use</a> a PermissionDescriptor returns the current permission state without user interaction (when the current state is not "prompt") or asks the user's permission.
-    In both cases, the permission to use is requested and either granted or denied.
-  </div></dd>
+    <div class="note">
+      It does not matter whether the algorithm to <a>request permission to use</a> a PermissionDescriptor returns the current permission state without user interaction (when the current state is not "prompt") or asks the user's permission.
+      In both cases, the permission to use is requested and either granted or denied.
```

Comment:
This sentence is not very clear to me.

Reply:
In both cases (when the algorithm returns the current permission state without user interaction or when the algoithm asks the user's permission), the permission to use is requested. Thus the first condition above stating that _the track has been created without requesting permission to use a PermissionDescriptor with its name member set to camera and its panTiltZoom member set to true_ is not fulfilled. 
In the both cases, the permission is either granted or denied. If it is granted, the second condition above stating that _that permission request is denied_ is not fulfilled, and if it is denied, the second condition above is fulfilled.

How would you change that note?

Reply:
@eehakkin Why is the note there? Is it to warn readers the [request permission to use](https://w3c.github.io/permissions/#request-permission-to-use) algorithm, despite its name, doesn't necessarily prompt the user every time?

If so, then lets remove the note, since that's intrinsic behavior of the permissions spec, and not unique to imageCapture. Other specs don't have such notes when invoking this algorithm.

Reply:
> @eehakkin Why is the note there? Is it to warn readers the [request permission to use](https://w3c.github.io/permissions/#request-permission-to-use) algorithm, despite its name, doesn't necessarily prompt the user every time?

Yes, for that reason, exactly. It was requested at https://github.com/w3c/mediacapture-image/pull/250.

Reply:
@eehakkin where exactly was it requested (can you deep-link)?

I don't think this spec needs a note more than any other spec, but if we keep it I suggest clearer language, e.g.:

_"Note: The **request permission to use** algorithm, despite its name, does not necessarily prompt the user every time."_

Reply:
Because this PR is not about that note, I created https://github.com/w3c/mediacapture-image/pull/266 for the removal.